awx/main/tasks/jobs.py

```diff
@@ -193,6 +194,13 @@ def build_execution_environment_params(self, instance, private_data_dir):
                 # Uppercase Z restricts access (in weird ways) to 1 container at a time
                 if this_path.count(':') == MAX_ISOLATED_PATH_COLON_DELIMITER:
                     src, dest, scontext = this_path.split(':')
+
+                    # scontext validation via performed via API, but since this can be overriden via settings.py
+                    # let's ensure scontext is one that we support
+                    if scontext not in CONTAINER_VOLUMES_MOUNT_TYPES:
```

Comment:
I dont think `scontext` is accurate. Should this be something like `mount_option`?

Reply:
You got it